recipes/glew/run_test.bat

```diff
@@ -0,0 +1,7 @@
+conda env create -f %RECIPE_DIR%\test\test-environment-windows.yml
```

Comment:
It appears the sundials doesn't solve this as they have commented all of the tests. I think this was missed in the review of xtensor.